lib/virtfs/block_io.rb

```diff
@@ -11,6 +11,8 @@ module VirtFS
   class BlockIO
     MIN_BLOCKS_TO_CACHE = 64
 
+    attr_reader :seek_pos
```

Comment:
@roliveri I'm not fully following, even the Ruby IO object provides access to the 'pos' (current seek pos pointer):

https://ruby-doc.org/core-2.2.2/IO.html#method-i-pos

Why would this matter for multithreading purposes? Are standard ruby IO objects thread safe?